src/app/shared/table/actions/users/table-create-payment-method-action.ts

```diff
@@ -24,6 +25,13 @@ export class TableCreatePaymentMethodAction extends TableCreateAction {
 
   private createPaymentMethod(paymentMethodDialogComponent: ComponentType<unknown>,
     dialog: MatDialog, dialogParams: DialogParams<PaymentDialogData>, refresh?: () => Observable<void>) {
-    super.create(paymentMethodDialogComponent, dialog, dialogParams, refresh);
+    super.create(paymentMethodDialogComponent, dialog, dialogParams, refresh, {
+      minWidth: ScreenSize.M,
+      maxWidth: ScreenSize.M,
+      width: ScreenSize.M,
+      minHeight: ScreenSize.XL,
+      maxHeight: ScreenSize.XL,
+      height: ScreenSize.XL
+    });
```

Comment:
I really do not like to introduce display dependent parameters for rendering pages, so can you confirm that they will not be used by Angular?